src/seedu/addressbook/parser/Parser.java

```diff
@@ -72,34 +62,36 @@ public Command parseCommand(String userInput) {
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
-
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
-
-        case DeleteCommand.COMMAND_WORD:
-            return prepareDelete(arguments);
-
-        case ClearCommand.COMMAND_WORD:
-            return new ClearCommand();
-
-        case FindCommand.COMMAND_WORD:
-            return prepareFind(arguments);
-
-        case ListCommand.COMMAND_WORD:
-            return new ListCommand();
-
-        case ViewCommand.COMMAND_WORD:
-            return prepareView(arguments);
-
-        case ViewAllCommand.COMMAND_WORD:
-            return prepareViewAll(arguments);
-
-        case ExitCommand.COMMAND_WORD:
-            return new ExitCommand();
-
-        case HelpCommand.COMMAND_WORD: // Fallthrough
-        default:
-            return new HelpCommand();
+            case AddCommand.COMMAND_WORD:
```

Comment:
Try to follow the indentation in the [coding standard](https://google.github.io/styleguide/javaguide.html#s4.8.4-switch) for switch statements (: